src/agents/__init__.py

```
@@ -3,5 +3,14 @@
+from .synthetic_worker import SyntheticWorkerAgent
+from .swarm_orchestrator import SwarmOrchestrator, SwarmStrategy, SubAgentResult
 
-__all__ = ['BaseAgent', 'AgentContext']
+__all__ = [
+    'BaseAgent',
+    'AgentContext',
+    'SyntheticWorkerAgent',
+    'SwarmOrchestrator',
+    'SwarmStrategy',
+    'SubAgentResult',
+]
```

Ruff (0.15.5)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)